### DISCUSSION TITLE
Does anybody try this in Google Cloud Run?

### DISCUSSION THREAD

**Discussion by inovikov78** [Rongronggg9/RSS-to-Telegram-Bot#221] [Q&A] [answered]
Does anybody try this in Google Cloud Run?

**Comment by Rongronggg9** [comment, marked as answer]
Google Cloud Run is a **serverless** platform.
RSS to Telegram Bot is a **serverful** bot.
So it is impossible.

**Reply by inovikov78** [reply]
Gotcha! Thank you!
